README.md

```diff
@@ -10,15 +10,15 @@ It made the children laugh and play,
 And so the Teacher turned him out,  
    But still he slithered near,  
 And waited patiently about,  
-   Till Mary did appear ;  
+   Till Walt did appear ;  
 And then he slid to her, and coiled  
    His body 'round her arm,  
 As if he said—‘ I’m not afraid—  
    You’ll keep me from all harm.’  
 
-‘ What makes the snake love Mary so ?’  
+‘ What makes the snake love Walt so ?’  
    The eager children cry—  
```

Comment:
Actually, it seems the author intended the whitespace artistically...

Reply:
Ohhhhh. Okay. it's still gonna bother me but I'm cool with it!